handlers/homepage/handler.go

```diff
@@ -0,0 +1,29 @@
+package homepage
+
+import (
+	"net/http"
+	"github.com/ONSdigital/dp-dd-frontend-controller/renderer"
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/dp-frontend-models/model/dd/homepage"
+)
+
+// Handler handles requests to the homepage
+func Handler(w http.ResponseWriter, req *http.Request) {
+	page := homepage.Homepage{}
+
+	body, err := renderer.Render(page, "dd/homepage")
+	if err != nil {
```

Comment:
Actually, in this case we are calling the renderer for what should be a known template, so a 404 is a misconfiguration of the system (we're trying to use a template that doesn't exist). I'll improve the error logging though.